ipmi_remote.yml

```diff
@@ -0,0 +1,45 @@
+# Configuration file for ipmi_exporter
+
+# This is an example config for scraping remote hosts via IPMI.
+# Information required to access remote IPMI interfaces can be supplied in the
+# 'modules' section. A scrape can request the usage of a given config by
+# setting the `module` URL parameter.
+
+modules:
+        default:
+                # These settings are used if no module is specified, the
+                # specified module doesn't exist, or of course if
+                # module=default is specified.
+                user: "default_user"
+                pass: "example_pw"
+                driver: "LAN_2_0"
+                privilege: "user"
+                # Available collectors are bmc, ipmi, and dcmi
+                collectors:
+                - bmc
+                - ipmi
+                # Got any sensors you don't care about? Add them here.
+                exclude_sensor_ids:
+                - 2
+                - 29
+                - 32
+                - 50
+                - 52
+                - 55
+        dcmi:
+                # Use these settings when scraped with module=dcmi.
+                user: "admin_user"
+                pass: "another_pw"
+                privilege: "admin"
+                driver: "LAN_2_0"
+                collectors:
+                - dcmi
+        10.8.0.2:
+                # If a scrape does NOT set the module parameter but the target
```

Comment:
IMO, modules should always be explicit. Less magic, more consistency.

Reply:
I'll think about it. A lot of people will probably end up having one module for every target, so I think there is a good amount of convenience in this. But besides that?

Reply:
@bitfehler and myself at first had the same reaction, i.e. "too much magic". But then we thought it's actually not _that_ much magic. It's really just the following: "If you specify no module explicitly, the fallback module name is the target parameter. If no such module exists, the `default` module is used as the final fallback." And it comes with a lot of convenience in the case @bitfehler mentioned.

Reply:
Yea, it's not that much. But I would prefer things follow the existing patterns laid out by blackbox_exporter and snmp_exporter. I need to think about this a bit.

Reply:
@SuperQ okay, I went with your suggestion. Since we broke existing configs anyways it was a good time to take a clean slate approach, so no more implicit target as module name. You can of course still do it explicitly (and likely want to for certain setups, so I added a note to the README).
I did decide to keep the notion of a "default" module, because both blackbox and snmp exporter have that as well. I think we'll go ahead and ship this now, further fine-tuning can always happen later on.